reformat.sh

```
@@ -6,12 +6,17 @@ echo ""
 echo "Formatting enterprise frontend..."
 pnpm exec prettier ../enterprise/frontend/src/ --write
 
+# Pin ruff to the version enforced by CI (.github/workflows/backend-linters.yaml)
+# and the pre-commit hook (.pre-commit-config.yaml). Using uvx keeps this aligned
+# regardless of the globally installed ruff version.
+RUFF="uvx ruff@0.15.17"
+
 echo ""
 echo "Formatting backend..."
 cd ../backend
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)